src/BridgeIHMCORS.cpp

```diff
@@ -126,13 +148,15 @@ double rad2deg(const double angleInRad)
 
 void BridgeIHMCORS::run()
 {
+  
     yarp::os::LockGuard guard(m_deviceMutex);
-
+    
```

Comment:
If possible, try to avoid trailing whitespaces (most editor has an option to automatically remove them).